File: src/main/java/com/elissamc/picturelogin/imgmsg/Testing.java
package com.elissamc.picturelogin.imgmsg;

public class Testing {



    public static char getChar() {
        String character = "block";
        if (character.equalsIgnoreCase("block")) {
            return ImageChar.BLOCK.getChar();
        }
        if (character.equalsIgnoreCase("dark_shade")) {
            return ImageChar.DARK_SHADE.getChar();
        }
        if (character.equalsIgnoreCase("medium_shade")) {
            return ImageChar.MEDIUM_SHADE.getChar();
        }
        if (character.equalsIgnoreCase("light_shade")) {
            return ImageChar.LIGHT_SHADE.getChar();
        }
        return ImageChar.BLOCK.getChar();
    }
}

File: src/main/java/com/elissamc/picturelogin/imgmsg/Testing.kt
package com.elissamc.picturelogin.imgmsg

object Testing {


    val char: Char
        get() {
            val character = "block"
            if (character.equals("block", ignoreCase = true)) {
                return ImageChar.BLOCK.char
            }
            if (character.equals("dark_shade", ignoreCase = true)) {
                return ImageChar.DARK_SHADE.char
            }
            if (character.equals("medium_shade", ignoreCase = true)) {
                return ImageChar.MEDIUM_SHADE.char
            }
            return if (character.equals("light_shade", ignoreCase = true)) {
                ImageChar.LIGHT_SHADE.char
            } else ImageChar.BLOCK.char
        }
}